policyengine_us/parameters/gov/states/al/dhs/income/earned.yaml

```diff
@@ -0,0 +1,13 @@
+description: AL TANF counts these income sources as earned income.
+values:
+  2022-01-01: 
+    - employment_income
+    - self_employment_income
+    - earned_income_tax_credit
+    deductions:
+      deduction_percentage: 0.20  # 20%
```

Comment:
we need to create a parameter file called `rate.yaml` - create a folder called `misc_exepenses` in the `income` folder

Reply:
the create a `al_tanf_work_expense_deduction.py` with the calculation for this which takes:

```
misc_expense = add(tax_unit, period, ["misc_deduction"])
        * the 20% rate parameter
```